library/channel.c

```diff
@@ -560,17 +578,17 @@ static void hello_reply_cb(void *ctx, message *msg) {
         ch->notify_cb(ch, EdgeRouterUnavailable, ch->notify_ctx);
     }
 
-    for (int i = 0; i < ch->conn_reqs_n; i++) {
-        struct ch_conn_req *r = ch->conn_reqs[i];
+    while (!LIST_EMPTY(&ch->conn_reqs)) {
+        struct ch_conn_req *r = LIST_FIRST(&ch->conn_reqs);
+        LIST_REMOVE(r, next);
         r->cb(ch, r->ctx, cb_code);
         free(r);
     }
-    ch->conn_reqs_n = 0;
 
     if (success) {
         // initial latency
         ch->latency = uv_now(ch->loop) - ch->latency;
-        uv_timer_start(&ch->timer, send_latency_probe, 0, 60 * 1000);
+        uv_timer_start(&ch->timer, send_latency_probe, LATENCY_INTERVAL, 0);
```

Comment:
just checking - i see the params are flopped... was this a bug that was fixed? just wanna make sure it was intentional

Reply:
on purpose